libqtile/layout/spiral.py

```diff
@@ -188,13 +189,19 @@ def configure(self, win, screen):
             bc = self.border_normal
 
         (x, y, w, h), margins = self._fix_double_margins(x, y, w, h)
+        num_windows = len(self.clients)
+
+        if num_windows == 1 and not self.border_on_single:
```

Comment:
```suggestion
        if len(self.clients) == 1 and not self.border_on_single:
```
Don't think we need a separate variable as you're only checking it once.

Reply:
Change request implemented.